Explorer/Assets/DCL/LOD/Settings/LODSettingsAsset.cs

```diff
@@ -15,6 +15,7 @@ public class LODSettingsAsset : ScriptableObject, ILODSettingsAsset
         };
 
         [field: SerializeField] public int SDK7LodThreshold { get; set; } = 2;
+        [field: SerializeField] public float TimeToChangeToLod { get; set; } = 5;
```

Comment:
Please write documentation or tooltip to provide some context about this new field, why is it necessary, what is its purpose or who uses it for what.